lib/gds_api/version.rb

```diff
@@ -1,3 +1,3 @@
 module GdsApi
-  VERSION = '26.3.1'
+  VERSION = '26.4.1'
```

Comment:
Shouldn't update the version here. This is probably why this PR has merge conflicts.